### PR TITLE
Fix path to Cmake toolchain files in hints

### PR DESCRIPTION
**Pull request by Keno**
Apparently this got renamed.

### CODE REVIEW

File: src/wizard/hints.jl [hunk 1]
```diff
@@ -66,7 +66,7 @@ function provide_hints(state::WizardState, path::AbstractString)
                 print(state.outs,   "    This file is likely input to CMake. ")
                 println(state.outs,   "The recommended options for CMake are")
                 println(state.outs)
-                printstyled(state.outs, "      cmake -DCMAKE_INSTALL_PREFIX=\$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/\$target/\$target.toolchain", bold=true)
+                printstyled(state.outs, "      cmake -DCMAKE_INSTALL_PREFIX=\$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/\$target/\$target.cmake", bold=true)
```
Comment (giordano) [inline src/wizard/hints.jl]: Well spotted (and I even modified this yesterday...), but this should rather use the environment variable `${CMAKE_TARGET_TOOLCHAIN}`:
```suggestion
                printstyled(state.outs, "      cmake -DCMAKE_INSTALL_PREFIX=\$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}", bold=true)
```

Comment (Keno) [reply src/wizard/hints.jl]: Confirmed working. Applied!

**Comment by Keno** [comment]
I'll let @staticfloat merge, but I assume the CI failure are just because the repo is in the process of being worked over.

**Comment by giordano** [comment]
Yes, tests are not quite ready.  He started to revamp them with https://github.com/JuliaPackaging/BinaryBuilder.jl/commit/440ce215a2f223083896ad4836d21a89cdcd461b

**Pull request merged**